setup.py

```diff
@@ -45,6 +45,7 @@ def run(self):
 MIN_PYTHON_VERSION = '>=3.6.*'
 
 requirements = [
+    'Pillow==6.2.1',
```

Comment:
@zhanghang1989  told me this should be: 'Pillow<=6.2.1',

Reply:
Added